### ISSUE TITLE
Specify custom pragmas to be ignored for line-too-long by the formatter 

### ISSUE THREAD

**Issue by getzze** [astral-sh/ruff#11941]
Some pragma comments (for instance "noqa", "type", "pylint") when put at the end of a long line do not trigger a reformatting of the line when running `ruff format`

```python
a = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"  # noqa: F401
```
with `line-length=80` this line does not get reformatted.

From what I could guess, the list of ignored pragmas is hard-coded in this file:
https://github.com/astral-sh/ruff/blob/main/crates/ruff_python_trivia/src/pragmas.rs

It would be great if we could specify our own pragmas to be ignored, for instance I would need to ignore `# pragma: no cover` and `# spellchecker`.

This line will be reformatted
```python
a = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"  # pragma: no cover
```

For the linter, there is the `task-tags` option in combination with `ignore-overlong-task-comments=true`, but the name is not explicit enough and it is ignored by the formatter anyway.

I currently have to add an ignored pragma before to prevent reformatting:

```python
a = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"  # pylint: line-too-long  # pragma: no cover
```

**Comment by MichaReiser** [comment]
Yeah I think I'm open to this kind of configuration as it doesn't change the formatting itself, it just allows customizing what the formatter considers a pragma
